MARC.js

```diff
@@ -465,6 +483,9 @@ record.prototype.translate = function(item) {
 					if (creatorFields[i] == "100" || creatorFields[i] == "700" ) {
 						creatorObject = ZU.cleanAuthor(authorTab[j]['a'], "author", true)
 					} else {
+						authorTab[j]['a'] = authorTab[j]['a'].replace(/^[\s\u00A0\.\,\/\[\]\:]+/, '')
```

Comment:
I copied this replacement from https://github.com/zotero/zotero/blob/4.0/chrome/content/zotero/xpcom/utilities.js#L162 and the last change there was performed by @aurimasv .